support-frontend/assets/pages/paper-subscription-landing/components/hero/campaignHeader.jsx

```diff
@@ -11,10 +11,25 @@ import SvgChevron from 'components/svgs/chevron';
 import GridImage from 'components/gridImage/gridImage';
 import { sendTrackingEventsOnClick } from 'helpers/subscriptions';
 import { getDiscountCopy } from './discountCopy';
-import './joyOfPrint.scss';
 import type { State } from 'pages/paper-subscription-landing/paperSubscriptionLandingPageReducer';
 import { getMaxSavingVsRetail } from 'helpers/productPrice/paperProductPrices';
 import type { DiscountCopy } from 'pages/paper-subscription-landing/components/hero/discountCopy';
+import {
+  campaignHeader,
+  campaignCopy,
+  heading,
+  headingLineOne,
+  headingLineTwo,
+  subheading,
+  graphicOuter,
+  graphicInner,
+  graphic,
+  badge,
+} from './campaignHeaderStyles';
+
+// We need to preserve this little bit of scss but hope to gradually move it out
```

Comment:
Why do we still need it?

Reply:
Because changing the product hero component was like pulling a giant thread in a tapestry. It felt like a much larger piece of work. Probably a bit like that checkout layout one.

Reply:
It would be nice to take some time and do it carefully when we have visual regression tests

Reply:
I have put in a [trello card](https://trello.com/c/d02heeJI/3151-refactor-productpagehero-component-so-it-works-with-emotion) for this work.